src/app/lib/config/services/pidgin.js

```diff
@@ -4,6 +4,7 @@ import { latin } from '@bbc/gel-foundations/scripts';
 import '@bbc/psammead-locales/moment/pcm';
 
 const pidgin = {
+  lang: 'pcm',
```

Comment:
We shouldn't be adding these values, there is already isoLang in these configs and they are all correct.

Reply:
isoLang is not available for the all the services ....its there for Igbo,yoruba and pidgin and not for any other languages.

Reply:
This page is live https://www.bbc.com/pidgin and is showing pcm in lang. This value is coming from somewhere in this config already, we must not add it again just for an e2e test. Live site config must never be changed to facilitate e2e testing.

Reply:
Speaking with the front page pod @aacn500 seems like these three services are missing the `lang` key in the config and currently live is fetching from a wrong place.

Reply:
Ok good resolution!